plutus-core/plutus-core/src/PlutusCore/Subst.hs

```diff
@@ -221,35 +223,45 @@ programMapNames f g (Program a v term) = Program a v (termMapNames f g term)
 -- Free variables
 
 -- | Get all the free term variables in a term.
-fvTerm :: Ord name => Traversal' (Term tyname name uni fun ann) name
+fvTerm :: HasUnique name unique => Traversal' (Term tyname name uni fun ann) name
```

Comment:
This is actually a better signature full stop. `Ord name` suggests it might work for e.g. de bruijn names, but it really won't.